<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_principal</class>
 <widget class="QMainWindow" name="vtn_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mi Primer GUI v1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_nombre">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre:*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_nombre">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ingrese su nombre</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apellido">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellido:*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_apellido">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_guardar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_email">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:*</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sexo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexo:*</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_sexo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar opci√≥n</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Masculino</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Femenino</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_cedula">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Cedula:*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cedula">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_limpiar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mnb_menu_principal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnu_archivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="mni_archivo"/>
    <addaction name="mni_abrir"/>
    <addaction name="mni_cerrar"/>
   </widget>
   <widget class="QMenu" name="mnu_ayuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="mni_acerca_de"/>
   </widget>
   <addaction name="mnu_archivo"/>
   <addaction name="mnu_ayuda"/>
  </widget>
  <widget class="QStatusBar" name="stb_estado"/>
  <action name="mni_archivo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="mni_abrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="mni_cerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="mni_acerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>txt_nombre</tabstop>
  <tabstop>txt_apellido</tabstop>
  <tabstop>txt_email</tabstop>
  <tabstop>txt_cedula</tabstop>
  <tabstop>cb_sexo</tabstop>
  <tabstop>btn_guardar</tabstop>
  <tabstop>btn_limpiar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
